server/src/main/java/org/apache/iotdb/db/conf/IoTDBConstant.java

```diff
@@ -65,4 +65,6 @@ private IoTDBConstant() {
   public static final String USER = "User";
   public static final String PRIVILEGE = "Privilege";
 
+  public static final int MEMTABLE_NUM_INCREMETN = 4;
```

Comment:
```suggestion
  public static final int MEMTABLE_NUM_INCREMENT = 4;
```

or  MEMTABLE_IN_EACH_STORAGE_GROUP